src/Symfony/Component/DependencyInjection/Argument/ServiceLocator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Argument;
+
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+
+/**
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class ServiceLocator
```

Comment:
> You're just looking for psr-11

PSR-11 is still about DI containers. It semantically couples the code with dependency injection container concept. This is odd: I don't want to know whether it is container or simple array-like map of "command handlers". DI container is just specific type of a map.